
xwGTK undef BOOL if ODBC is used.
Next version of xwGTK does not have db support.

---

diff -urNp truecrypt-5.0-source/Common/Crypto.h truecrypt-5.0-source.new/Common/Crypto.h
--- truecrypt-5.0-source/Common/Crypto.h	2008-02-04 19:15:38.000000000 +0200
+++ truecrypt-5.0-source.new/Common/Crypto.h	2008-02-10 22:40:55.000000000 +0200
@@ -29,6 +29,10 @@
 
 #include "Tcdefs.h"
 
+#ifndef BOOL
+#define BOOL BOOL1
+#endif
+
 #ifdef __cplusplus
 extern "C" {
 #endif
diff -urNp truecrypt-5.0-source/Makefile truecrypt-5.0-source.new/Makefile
--- truecrypt-5.0-source/Makefile	2008-02-04 20:19:02.000000000 +0200
+++ truecrypt-5.0-source.new/Makefile	2008-02-10 22:40:34.000000000 +0200
@@ -42,7 +42,7 @@ export CFLAGS := -W
 export CXXFLAGS := -Wall
 
 C_CXX_FLAGS := -MMD -I$(BASE_DIR) -I$(BASE_DIR)/Crypto
-C_CXX_FLAGS += -DBOOL=int -DFALSE=0 -DTRUE=1
+C_CXX_FLAGS += -DBOOL=int -DBOOL1=int -DFALSE=0 -DTRUE=1
 C_CXX_FLAGS += -D__int8=char -D__int16=short -D__int32=int '-D__int64=long long'  # Tested in PlatformTest
 
 export LFLAGS :=
